lib/mixlib/install/product.rb

```diff
@@ -162,6 +170,17 @@ def lookup(key, version = :latest)
         product.version(version)
         product
       end
+
+      def products_available_on_downloads_site
+        product_urls = {}
+
+        products.each do |product|
+          url = lookup(product).downloads_product_page_url
+          product_urls[product] = url if url != :not_available
```

Comment:
Can we just return the complete `Product` instances vs the download product page url only?

Should be as easy as returning a sub-hash:

```ruby
products.delete_if {|product_key, product| product.downloads_product_page_url == :not_available }
```